osu.Game/Updater/SimpleUpdateManager.cs

```diff
@@ -94,17 +94,23 @@ private string getBestUrl(GitHubRelease release)
                     break;
 
                 case RuntimeInfo.Platform.iOS:
-                    // iOS releases are available via testflight. this link seems to work well enough for now.
-                    // see https://stackoverflow.com/a/32960501
-                    return "itms-beta://beta.itunes.apple.com/v1/app/1447765923";
+                    if (release.Assets?.Exists(f => f.Name.EndsWith(".ipa", StringComparison.Ordinal)) == true)
```

Comment:
Not sure about relying on the existence of the `.ipa`. It's more likely to go away in the future than to stay.

See: https://github.com/ppy/osu/issues/23218#issuecomment-1510719457

Reply:
Hmm, I think it's okay to rely on this for the time being. At least it gives us a way to manage availability.